src/controllers/userController.ts

```diff
@@ -173,3 +175,118 @@ export const updateUserProfile = async (req: Request, res: Response) => {
         })
     }
 }
+
+
+export const resetPasswordEmail = async (req: Request, res: Response) => {
+    const { email } = req.body;
+
+    try {
+        const user = await findUserByEmail(email);
+
+        if (!user) {
+            return res.status(404).json({
```

Comment:
Can you please use the API_RESPONSE to return your response

Reply:
done